compiler/llvm/clangUtil.cpp

```diff
@@ -2184,10 +2184,34 @@ void runClang(const char* just_parse_filename) {
   clangOtherArgs.push_back("sys_basic.h");
 
   if (!just_parse_filename) {
+
+    if (localeUsesGPU()) {
+      //create a header file to include header files from the command line
+      std::string genHeaderFilename;
+      genHeaderFilename = genIntermediateFilename("generated_header.h");
+      FILE* fp =  openfile(genHeaderFilename.c_str(), "w");
+
+      std::string ifdefStrBegin("#ifdef __cplusplus\nextern \"C\" {\n#endif");
+      fprintf(fp, "%s\n", ifdefStrBegin.c_str());
+
+      int filenum = 0;
+      while (const char* inputFilename = nthFilename(filenum++)) {
+        if (isCHeader(inputFilename)) {
+          fprintf(fp, "%s%s%s\n", "#include <", inputFilename,">");
```

Comment:
AFAIK this should use `#include "header.h"` rather than `#include <header.h>` so that it can include things in the current directory e.g.. I think of `#include <header.h>` as just for including things in `/usr/include` but the `#include "header.h"` version handles `/usr/include` as well as the current directory.